pkg/cmd/cli/cmd/newapp.go

```diff
@@ -235,7 +235,7 @@ func setupAppConfig(f *clientcmd.Factory, c *cobra.Command, args []string, confi
 
 func setLabels(labels map[string]string, result *newcmd.AppResult) error {
 	if len(labels) == 0 {
-		return nil
+		labels = map[string]string{"app": result.Name}
```

Comment:
should likely check when result.Name is empty... no pipelines got created because you were only instantiating templates.